target/linux/ramips/dts/rt5350_zyxel_keenetic-lite_rev.b.dts

```diff
@@ -0,0 +1,121 @@
+/dts-v1/;
+
+#include "rt5350.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zyxel,keenetic-lite_rev.b", "ralink,rt5350-soc";
```

Comment:
I'd use "zyxel,keenetic-lite-b" here, and adjust everything else accordingly.

Reply:
renamed